front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx

```diff
@@ -25,6 +25,8 @@ import { getIsUpdating } from 'reducers/osrdsimulation/selectors';
 import { useAppDispatch } from 'store';
 
 const MAP_MIN_HEIGHT = 450;
+const SPEEDSPACECHART_HEIGHT = 521.5;
```

Comment:
I think it's easier to read like this ;)
```suggestion
const SPEED_SPACE_CHART_HEIGHT = 521.5;
```

Reply:
done